packages/trigger/src/index.ts

```diff
@@ -50,6 +50,11 @@ export class TriggerCache<T> {
     this.#map.get(key)?.$$();
   }
 
+  dirtyAll() {
```

Comment:
`dirtyAll` is needed to be able to clean all tracked keys. Otherwise the user has no access to keys and not able to clear them without own keys tracking.

This is was an issue for Set and Map implementation where we have iterated over currently defined keys not all tracked keys.